### PR TITLE
Provide disable props to all widget and pass disable props in babel view

### PR DESCRIPTION
**Pull request by iFlameing**

### CODE REVIEW

**Review by sneridagh — APPROVED** [review]
LGTM

**Review by ksuess — CHANGES_REQUESTED** [review]
Maybe one line of documentation: "On these conditions, these widgets need to be disabled. This can be done by …"

File: news/4396.feature [hunk 1]
```diff
@@ -0,0 +1 @@
+Provide disable props to all widget and pass disable props in babel view @iFlameing
```
Comment (stevepiercy) [inline news/4396.feature:1]:
```suggestion
Provide disabled props to all widgets, and pass disabled props in to babel views. @iFlameing
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]

**Comment by sneridagh** [comment]
@iFlameing please take a look at suggestions.

**Comment by iFlameing** [comment]
@sneridagh add the documentation as suggested.

**Pull request merged**